suite/src/utils/ceramicHelpers.ts

```diff
@@ -12,11 +12,11 @@ const did = new DID({ provider, resolver })
 export const metadata = { controllers: [] }
 
 export const newCeramic = async (apiUrl: string) => {
-    const ceramic = new CeramicClient(apiUrl, { syncInterval: 500 })
-    if (!did.authenticated) {
-        await did.authenticate();
-        (metadata.controllers as string[]) = [did.id]
-    }
-    await ceramic.setDID(did)
-    return ceramic
+  const ceramic = new CeramicClient(apiUrl, { syncInterval: 500 })
+  if (!did.authenticated) {
+    await did.authenticate()
+    ;(metadata.controllers as string[]) = [did.id]
```

Comment:
Huh, the semicolon at the start looks weird...

Reply:
I can't get rid of it.  The semicolon is required to avoid a compiler error, and prettier wants to put it at the start of this line not the end of the last one.  I think we'd need to change the ceramic eslint config to try to figure out how to fix that.  Maybe @ukstv knows if this is possible to fix.

But for now I'm going to go ahead with this as-is, this is just a quirk of prettier